ver/us/splat.yaml

```diff
@@ -107,7 +107,7 @@ segments:
     - [0x3BD40, hasm, os/osSetIntMask]
     - [0x3bde0, c, os/osCreatePiManager]
     - [0x3C160, c, os/osEPiWriteIo] # MOVE_ADDU
-    - [0x3c1c0, c, os/osEPiReadIo] # MOVE_ADDU
+    - [0x3c1c0, c, ultra/epiread]
```

Comment:
I would say we might wanna keep my `# MOVE_ADDU` comment, but I guess everything in the ultra/ dir will eventually all implicitly have the addu for move anyway...

I was using those comments to track which files had the difference, but I guess we can just move them into ultra/ and the new compiler will generate the correct code? This will also get rid of the need for the different asm macro right?

Reply:
Yeah, there's no need for the macros in those files anymore. Anything compiled with kmc gcc will use addu for moves, and in fact the macros probably wouldn't even work with kmc gcc as it seems to not have as robust support for inline asm (for example, INCLUDE_ASM doesn't work with it).